### DISCUSSION TITLE
Docker running the double v's

### DISCUSSION THREAD

**Discussion by Mar0xy** [owncast/owncast#3140] [General Owncast discussion]
I have noticed that for some reason when you run owncast in a docker container it tends to put the version as "vv0.1.0" (take note of the second v) I assume this wasn't planned as this also causes owncast to think there is an update available.

![image](https://github.com/owncast/owncast/assets/8841466/77bf0481-6cc2-4b45-9ca1-8a3e3d0b5bba)
![image](https://github.com/owncast/owncast/assets/8841466/e8e89cbe-f510-434c-9220-9b31d607a675)

**Comment by gabek** [comment]
Duplicate of https://github.com/owncast/owncast/issues/3066.
